lua/super-commit/buffer/open.lua

```diff
@@ -1,5 +1,11 @@
+local win_ids = require('super-commit/window/open').id
+
 local M = {}
+M.id = {}
```

Comment:
To be consistent, use the same variable name as possible.

This also applies to `win_ids`.

```suggestion
M.ids = {}
```

Reply:
Fixed it in ddaa0930d9a1d4ae025fa1a69c6d3fdddb1a7324